fdbserver/fdbserver.actor.cpp

```diff
@@ -887,94 +887,91 @@ void restoreRoleFilesHelper(std::string dirSrc, std::string dirToMove, std::stri
 	}
 }
 
-int main(int argc, char* argv[]) {
-	try {
-		platformInit();
-		initSignalSafeUnwind();
-
-#ifdef ALLOC_INSTRUMENTATION
-		g_extra_memory = new uint8_t[1000000];
-#endif
-		registerCrashHandler();
-
-		// Set default of line buffering standard out and error
-		setvbuf(stdout, NULL, _IOLBF, BUFSIZ);
-		setvbuf(stderr, NULL, _IOLBF, BUFSIZ);
+namespace {
+enum Role {
+	Simulation,
+	FDBD,
+	Test,
+	MultiTester,
+	SkipListTest,
+	SearchMutations,
+	DSLTest,
+	VersionedMapTest,
+	CreateTemplateDatabase,
+	NetworkTestClient,
+	NetworkTestServer,
+	Restore,
+	KVFileIntegrityCheck,
+	KVFileGenerateIOLogChecksums,
+	ConsistencyCheck
+};
+struct CLIOptions {
+	std::string commandLine;
+	std::string fileSystemPath = "", dataFolder, connFile = "", seedConnFile = "", seedConnString = "", logFolder = ".",
```

Comment:
We don't have to `= ""` as string is empty by default.

Reply:
Done